x/concentrated-liquidity/lp_test.go

```diff
@@ -17,21 +17,23 @@ func (s *KeeperTestSuite) TestMint() {
 	currentTick := sdk.NewInt(85176)
 	lowerTick := int64(84222)
 	upperTick := int64(86129)
-	liquidity, err := sdk.NewDecFromStr("1517.882323")
```

Comment:
why move this testing params from test to keeper layer?

Reply:
API has changed. This is the final API that we are going to have